pkg/storage/kv/etcd_kv.go

```diff
@@ -77,11 +77,17 @@ func (kv *etcdKVBase) LoadRange(key, endKey string, limit int) ([]string, []stri
 	// As a result, when we try to scan from "foo/", it ends up scanning from "/pd/foo"
 	// internally, and returns unexpected keys such as "foo_bar/baz".
 	key = strings.Join([]string{kv.rootPath, key}, "/")
-	endKey = strings.Join([]string{kv.rootPath, endKey}, "/")
+	var OpOption []clientv3.OpOption
+	// If endKey is "\x00", it means to scan with prefix.
+	if endKey == "\x00" {
```

Comment:
What's this change for?

Reply:
We don't rely on the root path to generate the key before calling `LoadTimestamp`, so for global TSO, the key will be `""` and  `GetPrefixRangeEnd` is no longer effective and return a zero byte. We need to handle this situation at the kv level to add a prefix option.